cue.yaml

```diff
@@ -0,0 +1,73 @@
+package:
+  name: cue
+  version: 0.4.3
+  epoch: 0
+  description: The home of the CUE language! Validate and define text-based and dynamic configuration
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/cue-lang/cue/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 3d51f780f6d606a0341a5321b66e7d80bd54c294073c0d381e2ed96a3ae07c6e
+  - runs: |
+      CGO_ENABLED=0 go build \
+        -trimpath -ldflags "-s -w -buildid= -X cuelang.org/go/cmd/cue/cmd.version=${{package.version}}" \
+        -o cue-cli ./cmd/cue
+      mkdir -p ${{targets.destdir}}/usr/share/bash-completion/completions 
+      mkdir -p ${{targets.destdir}}/usr/share/zsh/site-functions
+      mkdir -p ${{targets.destdir}}/usr/share/fish/vendor_completions.d
+      ./cue-cli completion bash > ${{targets.destdir}}/usr/share/bash-completion/completions/cue
+      ./cue-cli completion zsh > ${{targets.destdir}}/usr/share/zsh/site-functions/_cue
+      ./cue-cli completion fish > ${{targets.destdir}}/usr/share/fish/vendor_completions.d/cue.fish
+      install -m755 -D ./cue-cli "${{targets.destdir}}"/usr/bin/cue
+     
+update:
+  enabled: true
+  github:
+    identifier: cue-lang/cue
+    strip-prefix: v
+
+subpackages:
+  - name: cue-bash-completion
+    description: bash completion for cue
+    pipeline:
+      - runs: |
+          set -x
+          mkdir -p "${{targets.subpkgdir}}/usr/share/bash-completion"
+          mv "${{targets.destdir}}/usr/share/bash-completion/completions" \
```

Comment:
In addition to the suggested edit, these should run `./cue-cli completion foo > ${{targets.subpkgdir}}/usr/share/...`